File: Python Language/Q28.py
# Program to compute the discriminant

a = float(input("Enter value for a: "))
b = float(input("Enter value for b: "))
c = float(input("Enter value for c: "))

# Formula: disc = b^2 - 4ac
# In Python, ** is the exponent operator
disc = (b ** 2) - (4 * a * c)

print(f"The discriminant is: {disc}")

File: C Language/Q28.c
#include <stdio.h>

int main() {
    int a, b, c, disc;

    printf("Enter values for a, b, and c: ");
    scanf("%d %d %d", &a, &b, &c);

    disc = (b * b) - (4 * a * c);

    printf("The discriminant is: %d\n", disc);

    return 0;
}